src/overseer/config.clj

```diff
@@ -3,30 +3,41 @@
   optional, and Overseer should 'just work' out of the box. The entire set
   of configuration options are:
 
-    :store - Required backend store type to connect to. For now, must be :datomic
+    :store - Required backend store type to connect to. One of :datomic, :jdbc
```

Comment:
It is sort of odd, I was unsatisfied with the options here. I don't like having to explain alternatives, like you could define config as:

```
store:
  type: datomic
  config:
    uri: ...
```

But then you have to explain, if datomic then config is a map of uri. elsif jdbc, it's a map of ... . You know? I agree this needs revisiting though. Will think about it.